File: MISC/ANAS/java/01/Employee.java
public class Employee {

    private String name;
    private static int employeeCount = 0;

    public Employee(){
        employeeCount++;
    }

    public Employee(String name){
        this.name = name;
        employeeCount++;
    }

    public String getName(){
        return name;
    }

    public void setName(String name){
        this.name = name;
    }

    public static int getEmployeeCount(){
        return employeeCount;
    }

}

File: MISC/ANAS/java/01/employee.py

class Employee():

    employee_count = 0

    def __init__(self, name=None):
        Employee.employee_count += 1
        self.name = name

    def get_name(self):
        return self.name
    
    def set_name(self, name):
        self.name = name

    @staticmethod
    def get_employee_count():
        return Employee.employee_count
    

